src/containers/aspect/AspectDashboardContainer/AspectDashboardContainer.tsx

```diff
@@ -102,6 +96,15 @@ const AspectDashboardContainer: FC<AspectDashboardContainerProps> = ({
   const loading = hubLoading || challengeLoading || opportunityLoading;
   const error = hubError ?? challengeError ?? opportunityError;
 
+  const { data: creatorData, loading: loadingCreator } = useAspectCreatorQuery({
+    variables: { userId: aspect?.createdBy ?? '' },
+    skip: !aspect,
+  });
+  const creator = creatorData?.user;
+  const creatorAvatar = creator?.profile?.avatar?.uri;
+  const creatorName = creator?.displayName;
+  const createdDate = aspect && new Date(aspect.createdDate);
```

Comment:
I would avoid to pass rich objects as props if they could be instantiated in the consuming view. If there's absolute need for that, you have to wrap its instantiation in `useMemo()`.